tests/network/primary_pod_network.go

```diff
@@ -120,7 +119,13 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 				var vmi *v1.VirtualMachineInstance
 
 				BeforeEach(func() {
-					vmi = setupVMI(virtClient, vmiWithBridgeBinding())
+					vmi = setupVMI(
+						virtClient,
+						libvmi.NewAlpine(
+							libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding(libvmi.DefaultInterfaceName)),
+							libvmi.WithNetwork(v1.DefaultPodNetwork()),
```

Comment:
nit: can we have the same order for `WithInterface` and `WithNetwork`at both `setupVMI` instances ?

Reply:
Done